test/e2e/build_kubectl_buildkit_test.go

```diff
@@ -104,3 +109,16 @@ spec:
 		t.Fatalf("Expected >>>%s<<< to match >>>%s<<<", out, expectedOut)
 	}
 }
+
+func createBuilder(t *testing.T) {
```

Comment:
It might be helpful to create a `TODO` note here stating that this should be removed if/when it can be addressed upstream so it's not forgotten about.

Reply:
Resolving this comment. A separate issue will be opened regarding potentially fixing this upstream.

Reply:
i would still prefer to add a comment to indicate why this exists and link it to github issue in buildkit repo.

Reply:
Added the comment